src/Adapter/MySQL.php

```diff
@@ -299,12 +299,12 @@ protected function getFieldMapping()
                 'dependencyField' => 'nleft',
             ],
             'sales' => [
-                'tableName' => 'product_sale',
+                'tableName' => 'product',
                 'tableAlias' => 'psales',
-                'fieldName' => 'quantity',
+                'fieldName' => 'cantidad_vendida',
```

Comment:
I am pretty sure this might work for you but this is not working for the generic usecase